apiserver/pkg/util/job.go

```diff
@@ -52,9 +48,19 @@ func NewRayJob(apiJob *api.RayJob, computeTemplateMap map[string]*api.ComputeTem
 		cpus := "1"
 		memorys := "1Gi"
 		if apiJob.JobSubmitter.Cpu != "" {
+			// Ensure that CPU size is formatted correctly
+			_, err := resource.ParseQuantity(apiJob.JobSubmitter.Cpu)
+			if err != nil {
+				return nil, errors.New("cpu for ray job submitter is not specified correctly")
+			}
 			cpus = apiJob.JobSubmitter.Cpu
 		}
 		if apiJob.JobSubmitter.Memory != "" {
+			// Ensure that memory size is formatted correctly
+			_, err := resource.ParseQuantity(apiJob.JobSubmitter.Memory)
+			if err != nil {
+				return nil, errors.New("memory for ray job submitter is not specified correctly")
```

Comment:
why to wrap when creating the new error?
Something like this
```golang
return nil, fmt.Errorf("memory for ray job submitter is not specified correctly: %w", err
```

Reply:
fixed